glide.yaml

```diff
@@ -32,4 +32,6 @@ import:
   version: ~1.1
 - package: github.com/arschles/assert
   version: 6882f85ccdc7c1822b146d1a6b0c2c48f91b5140
-- package: github.com/minio/minio-go
+- package: github.com/docker/distribution
+  repo: https://github.com/kmala/distribution
```

Comment:
I would be using that once i get this merged https://github.com/deis/distribution/pull/7/files